server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterHealthAction.java

```diff
@@ -36,28 +37,7 @@
 @ServerlessScope(Scope.INTERNAL)
 public class RestClusterHealthAction extends BaseRestHandler {
 
-    @Override
-    public List<Route> routes() {
-        return List.of(new Route(GET, "/_cluster/health"), new Route(GET, "/_cluster/health/{index}"));
-    }
```

Comment:
Can you revert all the changes here that aren't necessary? I don't see a strong reason to move these things around, so then I'd rather keep them as is. It looks like the only thing you really changed is adding the `supportedCapabilities` method, right?

Reply:
Sure. My IDE got a bit enthusiastic with a reformat and applied the default "Static methods go at the top" rule to this class but I agree with you and @gmarouli that in the case of rest actions, it makes sense to have the route at the top.